plugin-locationlayer/src/main/java/com/mapbox/mapboxsdk/plugins/locationlayer/LocationLayerPlugin.java

```diff
@@ -496,6 +504,8 @@ private void setNavigationLayerVisibility(boolean visible) {
   @Override
   public void onCameraMove() {
     locationLayer.updateAccuracyRadius(location);
+    locationLayer.updateForegroundOffset(mapboxMap.getCameraPosition().tilt);
+    locationLayer.updateForegroundBearing((float) mapboxMap.getCameraPosition().bearing);
```

Comment:
reuse the CameraPosition object, now we are calling twice into c++ code while we can get away with just once.